bee-protocol/src/worker/milestone_cone_updater.rs

```diff
@@ -74,21 +74,22 @@ async fn update_messages_referenced_by_milestone<N: Node>(
 ) where
     N::Backend: StorageBackend,
 {
-    let mut to_visit = vec![message_id];
-    let mut visited = HashSet::new();
+    let mut parents_to_visit = vec![message_id];
```

Comment:
obviously the milestone we start with doesn't need to be visited. So initialize this vec with its 2 parents.

Reply:
the `cone_index` currently does not get set in tangle::add_milestone() so this would need to be changed too

Reply:
with the latest commit this is changed